Upgrade to 0.25.1 import styles and added dependencies to package.json

ptomasroos opened the pull request.

ptomasroos committed:
Updated importstyle accorind go to react-native 0.25.1

ptomasroos committed:
added proper dependencies to package.json

rturk commented:
There is no need to add RN26-RC to package.json. Adding just RN25 to package.json should work

ptomasroos committed:
Changed deps to be stable react-native

Changed react-native to  0.25.1 and react to 0.14.5 according to 0.25.1 release from RN.

ptomasroos commented:
Updated it now to 0.25.1 instead

ptomasroos committed:
Fixed trailing commas which might cause problems for certain setups

ptomasroos committed:
Changed to peerDependencies instead

ptomasroos committed:
Added react v 15* as well

ptomasroos committed:
Fixed double quote

ptomasroos committed:
Allow 0.26.* releases as well

FaridSafi commented:
Thank you, 0.0.5 has been published
